### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by isdampe**
Yo Phil,

This PR:

* Adds a basic gulp-based build system to the site
* Moves CSS to SCSS (integrated in the build system)
* Moves raw HTML files from the root directory to `src/`
* Has a little static file builder so we can reuse the header / footer HTML, etc.
* Has a little JS builder attached
* Has updated stylesheets to make the site responsive on tablets and mobiles
* Updates the README.md file to give some basic guidance on building and running new system

Let me know if you want to change anything

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,39 @@ To see the site live, visit https://philipcastiglione.github.io/deakin-sec-websi
...
+
+Open a terminal, and follow the instructions below.
+
+```bash
+cd /path/to/code
+git clone https://github.com/deakinsec/deakin-sec-website.git
+cd deakin-sec-website
+npm install
+```
+
```
Comment (PhilipCastiglione) [inline README.md]: Probably also want to include the gulp install instructions (eg from https://gulpjs.com/) since that's required for the next step.

**Comment by PhilipCastiglione** [comment]
Awesome work, the responsivity looks great!

File: README.md (hunk 1 above)
Comment (isdampe) [reply README.md]: Added in bcd8f8e

**Comment by isdampe** [comment]
Not sure if you saw @PhilipCastiglione I added bcd8f8e

**Review by PhilipCastiglione — APPROVED** [review]

**Pull request merged**